h/services/annotation_metadata.py

```diff
@@ -0,0 +1,51 @@
+import json
+
+from sqlalchemy import func, select
+from sqlalchemy.dialects.postgresql import insert
+from sqlalchemy.orm import Session
+
+from h.models import Annotation, AnnotationMetadata, AnnotationSlim
+from h.security import decrypt_jwe_dict
+
+
+class AnnotationMetadataService:
+    def __init__(self, db: Session, secret: str):
+        self._db = db
+        self._secret = secret
+
+    def set_annotation_metadata_from_jwe(self, annotation: Annotation, jwe: str):
+        """
+        Set the decrypted `jwe` dict as the metadata for `annotation`.
```

Comment:
```suggestion
        Decrypt `jwe` and use it to set `annotation`'s metadata.
```

"Set the decrypted `jwe` dict..." sounded to me like the `jwe` argument was supposed to already be a decrypted dict.